src/boot/bootctl.c

```diff
@@ -311,13 +311,15 @@ static int status_variables(void) {
 static int boot_entry_show(const BootEntry *e, bool show_as_default) {
         assert(e);
 
-        printf("        title: %s%s%s%s%s%s\n",
+        printf("        title: %s%s%s%s%s%s\n"
+               "         type: %s\n",
```

Comment:
hmm, yeah, i tried to come up with a better name, but I always came back to just calling them "conf", since that's also the suffix of the boot loader entry type 1 files. (maybe we should have called them .entry, or .boot or so, but i guess that's too late now)